solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -168,7 +183,7 @@ const fetchBondingData = async (web3Context) => {
     const sortitionPoolAddress = await fetchSortitionPoolForTbtc(web3Context)
     const createdBonds = await fetchCreatedBonds(
       web3Context,
-      operators,
+      operators.map((_) => _[0]),
```

Comment:
If we return a Map from `fetchOperatorsOf`, we could `Array.from(operators.keys())`.